consensus/src/main/kotlin/maru/consensus/qbft/QbftBlockCreatorFactory.kt

```diff
@@ -34,34 +35,30 @@ class QbftBlockCreatorFactory(
   private val proposerSelector: ProposerSelector,
   private val validatorProvider: ValidatorProvider,
   private val beaconChain: BeaconChain,
-  private val finalizationStateProvider: (BeaconBlockHeader) -> FinalizationState,
+  private val finalizationStateProvider: (BeaconBlockBody) -> FinalizationState,
   private val blockBuilderIdentity: Validator,
   private val eagerQbftBlockCreatorConfig: EagerQbftBlockCreator.Config,
-  private val metadataProvider: () -> BlockMetadata,
+  private val nextBlockTimestampProvider: NextBlockTimestampProvider,
+  private val clock: Clock,
 ) : QbftBlockCreatorFactory {
   override fun create(round: Int): BesuQbftBlockCreator {
-    require(round >= 0) {
-      "round must not be negative!"
-    }
-    val mainBlockCreator =
```

Comment:
I've created an issue https://github.com/Consensys/maru/issues/87 so we don't forget to fix the `DelayedQbftBlockCreator`